sklearn/feature_extraction/text.py

```diff
@@ -1318,6 +1307,17 @@ def fit_transform(self, raw_documents, y=None):
         min_df = self.min_df
         max_features = self.max_features
 
+        if self.fixed_vocabulary_ and self.lowercase:
```

Comment:
I considered placing this check in  `_validate_vocabulary`, but the method is called here:

https://github.com/scikit-learn/scikit-learn/blob/31c66a9c09e7f7d3de262cd5790de44608097850/sklearn/feature_extraction/text.py#L2007-L2010

and I do not think it makes sense to warn here. (**No action required**)

Reply:
Yes I thought the same